dbt/README.md

```diff
@@ -8,10 +8,10 @@ This directory stores the configuration for building our data catalog using
 ### In this document
 
 * [🖼️ Background: What does the data catalog do?](#%EF%B8%8F-background-what-does-the-data-catalog-do)
-* [🔨 How to rebuild models using GitHub Actions](#-how-to-rebuild-models-using-github-actions)
 * [💻 How to develop the catalog](#-how-to-develop-the-catalog)
 * [➕ How to add a new model](#-how-to-add-a-new-model)
-* [📝 How to add and run tests](#-how-to-add-and-run-tests)
+* [🔨 How to rebuild models using GitHub Actions](#-how-to-rebuild-models-using-github-actions)
+* [🧪 How to add and run QC tests and reports](#-how-to-add-and-run-qc-tests-and-reports)
```

Comment:
Again, for the sake of consistency and my own expectation as a reader I switched to using the 🧪 emoji here for QC tests and reports.